packages/augur-ui/src/modules/reporting/common.styles.less

```diff
@@ -1,5 +1,61 @@
 @import (reference) '~assets/styles/shared';
 
+.closeableNotice {
+  margin: @size-16 0;
+
+  > div {
+    align-items: center;
+    background: @color-closeable-notice-background;
+    border: 1px solid @color-outcome-two;
+    border-left-width: 0.25rem;
+    border-radius: @size-6;
+    display: flex;
+    font-family: Roboto Mono;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: bold;
+    justify-content: flex-start;
+    line-height: 12px;
```

Comment:
should use the @size-12 constant here and also for the .25 rem above

Reply:
Oops, yeah, I meant to use rem here.